source/globalCommands.py

```diff
@@ -2008,18 +2023,15 @@ def script_review_currentSymbol(self,gesture):
 	@script(
 		description=_(
 			# Translators: Input help mode message for toggle speech mode command.

```

Comment:
Shouldn't this now be "cycle speech modes command", or "Switch to next speech mode command"? It is no longer a toggle.

```suggestion
			# Translators: Input help mode message for switch to next speech mode command.
```